sqladmin/templates/sqladmin/list.html

```diff
@@ -198,7 +198,7 @@ <h3 class="card-title">{{ model_view.name_plural }}</h3>
         </a>
         <div class="dropdown-menu">
           {% for page_size_option in model_view.page_size_options %}
-          <a class="dropdown-item" href="{{ request.url.include_query_params(pageSize=page_size_option) }}">
+          <a class="dropdown-item" href="{{ request.url.include_query_params(pageSize=page_size_option,page=((pagination.page - 1) * pagination.page_size // page_size_option + 1))}}">
```

Comment:
Thank you for the PR.
I think it's better to update the pagination logic here: https://github.com/aminalaee/sqladmin/blob/main/sqladmin/pagination.py

Which makes it easier to test and also re-usable.